daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -177,7 +177,7 @@ class CLIConf(arguments: Array[String], stdout: PrintStream, stderr: PrintStream
           (name, schema.map(_.uri).toOption)
         case _ =>
           throw new Exception(
-            "Unrecognized Validator name %s.  Must be 'xerces', 'limited', 'off', or name of spi validator."
+            "Missing Validator name %s.  Must be 'xerces', 'daffodil', 'schematron', 'off', or name of a custom validator."
```

Comment:
If the default case is just when you have `--validate` with no name, then I think the %s will just be the empty string? Should this just be, "Missing validator name. Must be ..."